ui/imports/shared/panels/ProfileBioSocialsPanel.qml

```diff
@@ -84,13 +85,27 @@ Control {
     contentItem: ColumnLayout {
         spacing: 20
 
-        StatusBaseText {
+        StatusScrollView {
+            id: scrollView
+            visible: root.bio
+            padding: 0
+            rightPadding: scrollBar.visible ? 16 : 0
+            Layout.maximumHeight: 108
             Layout.fillWidth: true
-            visible: text !== ""
-            text: root.bio
-            wrapMode: Text.Wrap
-            font.weight: Font.Medium
-            lineHeight: 1.2
+            contentHeight: bioText.height
+            ScrollBar.vertical: StatusScrollBar {
+                id: scrollBar
+                policy: bioText.height > scrollView.availableHeight ? ScrollBar.AlwaysOn : ScrollBar.AlwaysOff
```

Comment:
It's supposed to be working out of the box with `ScrollBar.AsNeeded`. If its not working is component bug (probably mine):
https://github.com/status-im/status-desktop/blob/master/ui/StatusQ/src/StatusQ/Core/StatusScrollView.qml#L65
https://github.com/status-im/status-desktop/blob/master/ui/StatusQ/src/StatusQ/Controls/StatusScrollBar.qml#L30

Reply:
The problem is that scrollbar is not visible if content requires scrolling until you start scrolling.
After scrolling stops - the scrollbar disappears. I force it to be visible, if we have scrollable content, in order user understood, that there are more lines of Bio

Reply:
Got it, no questions :)